src/main/java/org/selenium/driver/RemoteWDriver.java

```diff
@@ -0,0 +1,37 @@
+package org.selenium.driver;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+
+public class RemoteWDriver implements Strategy {
+    private static DesiredCapabilities dc;
```

Comment:
I don't see the reason to have it as class variable.

Reply:
Fix it